### PR TITLE
Fix changelog of v0.27.0 release

### PR DESCRIPTION
**Pull request by tomek0123456789**
Closes #

## Introduced changes

-

## Checklist

- [ ] Linked relevant issue
- [ ] Updated relevant documentation
- [ ] Added relevant tests
- [ ] Performed self-review of the code
- [ ] Added changes to `CHANGELOG.md`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,23 +11,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-
-- `spy_messages_to_l1()` for listening in on messages to L1 sent by your contracts. [Read more here](https://foundry-rs.github.io/starknet-foundry/testing/testing-messages-to-l1.html).
-
 - `verify` subcommand to verify contract (walnut APIs supported as of this version). [Read more here](https://foundry-rs.github.io/starknet-foundry/appendix/sncast/verify.html)
 - support for v3 transactions on account deploy, deploy, declare, invoke
 - Newest class hash for OpenZeppelin account contracts
 - `account list` subcommand for listing all available accounts [Read more here](https://foundry-rs.github.io/starknet-foundry/appendix/sncast/account/list.html)
 
 #### Changed
-
```
Comment (piotmag769) [inline CHANGELOG.md:30]: Why remove this?

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,23 +11,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 - Renamed global cheatcodes listed [here](https://foundry-rs.github.io/starknet-foundry/appendix/cheatcodes.html) - cheatcode invocations affecting the global scope and working indefinitely, already marked with a `_global` suffix, received a `start_` prefix
-- Updated `cheat_account_contract_address` cheatcode - read more [here](https://foundry-rs.github.io/starknet-foundry/appendix/cheatcodes/account_contract_address.html) - added a `start_` prefix to the `..._global` invocation which works globally and indefinitely
+- Updated `cheat_account_contract_address` cheatcode - read more [here](https://foundry-rs.github.io/starknet-foundry/appendix/cheatcodes/account_contract_address.html)
 
 ### Cast
 
 #### Added
-
-- `spy_messages_to_l1()` for listening in on messages to L1 sent by your contracts. [Read more here](https://foundry-rs.github.io/starknet-foundry/testing/testing-messages-to-l1.html).
-
```
Comment (piotmag769) [inline CHANGELOG.md:23]: An additional newline is missing now

File: CHANGELOG.md [hunk 3]
```diff
@@ -11,23 +11,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Forge
 
+#### Added
+- `spy_messages_to_l1()` for listening in on messages to L1 sent by your contracts. [Read more here](https://foundry-rs.github.io/starknet-foundry/testing/testing-messages-to-l1.html).
+
 #### Changed
 - Renamed global cheatcodes listed [here](https://foundry-rs.github.io/starknet-foundry/appendix/cheatcodes.html) - cheatcode invocations affecting the global scope and working indefinitely, already marked with a `_global` suffix, received a `start_` prefix
-- Updated `cheat_account_contract_address` cheatcode - read more [here](https://foundry-rs.github.io/starknet-foundry/appendix/cheatcodes/account_contract_address.html) - added a `start_` prefix to the `..._global` invocation which works globally and indefinitely
+- Updated `cheat_account_contract_address` cheatcode - read more [here](https://foundry-rs.github.io/starknet-foundry/appendix/cheatcodes/account_contract_address.html)
```
Comment (piotmag769) [inline CHANGELOG.md:19]: I think the feature was there, just the docs wasn't, right? cc @integraledelebesgue 
If so document it here correctly

**Review by piotmag769 — COMMENTED** [review]
Changelog

File: CHANGELOG.md (hunk 1 above)
Comment (tomek0123456789) [reply CHANGELOG.md]: Unified all headers, all of them should have a newline below them now.

File: CHANGELOG.md (hunk 3 above)
Comment (tomek0123456789) [reply CHANGELOG.md]: Removed that since it is related to the line above (_Renaed global cheatcodes..._)

**Review by piotmag769 — APPROVED** [review]

**Review by cptartur — APPROVED** [review]

**Pull request merged**